onnxruntime/__init__.py

```diff
@@ -77,79 +77,100 @@
 
 onnxruntime_validation.check_distro_info()
 
+
+def check_and_load_cuda_libs(root_directory, cuda_libs):
+    # Convert the target library names to lowercase for case-insensitive comparison
```

Comment:
cuda_libs has not been converted to lower case yet.

Reply:
It is converted in line 173, but I can move it to inside of this function.